labels.yml

```diff
@@ -16,20 +16,40 @@ labels:
     color: 'f5f5f5'
 
   - name: monitoring
-    description: "This issue should be solved, but it should be monitored for a period before being closed"
+    description: "This issue should be solved, but let's keep an eye on it for a while before we close it."
     color: '357399'
 
   - name: performance
-    description:
+    description: "This issue significantly affects current performance, or will when we scale."
     color: '208116'
 
+  - name: operations
+    description: Anything related to our internal IT systems.
+    color: '262626'
+
+  - name: infrastructure
```

Comment:
i think `operations` and `infrastructure` can be merged into just `operations`. They greatly overlap. For example, you list DNSimple as operations, but it could just as easily be `infrastructure`. We could potentially have another label like `internal_tooling` to cover boxen and GDrive, but internal tooling also falls under the umbrella of operations. I think we could do with just `operations` for now, and break it into subcategories eventually. The goal for me is if an ops person joins, they can immediately filter issues relevant to them, and internal tooling would be in their AOR.